docs/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const copy = require('rollup-plugin-copy');
+const { createMpaConfig } = require('./_building-rollup/createMpaConfig.js');
+
+module.exports = async () => {
+  const mpaConfig = await createMpaConfig({
+    outputDir: '_site',
+    legacyBuild: true,
+    inputGlob: '_site-dev/**/*.html',
+    rootPath: path.resolve('_site-dev'),
+
+    // development mode creates a non-minified build for debugging or development
+    developmentMode: false, // process.env.ROLLUP_WATCH === 'true',
+
+    // set to true to inject the service worker registration into your index.html
+    injectServiceWorker: false,
+    workbox: false,
+  });
+
+  const dest = '_site/';
+  mpaConfig.plugins.push(
```

Comment:
Oooh, this is nice. Can't wait to steal this into the similar implementation I've been working no in Spectrum Web Components.